### PR TITLE
[release-2.3] Updated file formatting with prettier

### PR DESCRIPTION
**Pull request by dislbenn**
PR for fixing the linting job for the PR: https://github.com/stolostron/search-e2e-test/pull/177.

### CODE REVIEW

File: build/rbac-setup.sh [hunk 1]
```diff
@@ -11,7 +11,6 @@
 # DIR="$( find $HOME -type d -name "search-e2e-test" )"
 # using abosolute path, relative path is not working in travis
 
-set -e
```
Comment (jlpadilla) [inline build/rbac-setup.sh:14]: Add an `echo` message to remind us that this is disabled.

File: tests/config/rbac_yaml/kustomization.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 resources:
-- search-e2e-rbac-namespace.yaml
-- search-e2e-rbac-group.yaml
-- search-e2e-rbac-clusterrolebinding.yaml
-- search-e2e-rbac-rolebinding.yaml
+  - search-e2e-rbac-namespace.yaml
+  - search-e2e-rbac-group.yaml
+  - search-e2e-rbac-clusterrolebinding.yaml
+  - search-e2e-rbac-rolebinding.yaml
```
Comment (jlpadilla) [inline tests/config/rbac_yaml/kustomization.yaml:5]: Spaces are important in yaml files, so please double check all changes to yaml files.  If needed you can disable yaml from prettier.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

File: tests/config/rbac_yaml/kustomization.yaml (hunk 2 above)
Comment (dislbenn) [reply tests/config/rbac_yaml/kustomization.yaml:5]: Correct, this was just adding spaces to the array elements within the YAML file.

Comment (dislbenn) [reply tests/config/rbac_yaml/kustomization.yaml:5]: For now, I believe that this should be okay; however, if there are issues in the future, I will disable the yamls from prettier.

File: build/rbac-setup.sh (hunk 1 above)
Comment (dislbenn) [reply build/rbac-setup.sh:14]: Readding this error flag.

**Pull request merged**